Sources/PaymentsCore/GetClientIDResponse.swift

```diff
@@ -0,0 +1,4 @@
+struct GetClientIDResponse: Decodable {
+
+    let clientId: String
```

Comment:
```suggestion
    let clientID: String
```

Reply:
Yeah this one's interesting I can't find a way to get it to work with `Decodable`. The JSON property name is `client_id` and we use `.convertFromSnakeCase` in the decoder.

Reply:
Ah, forgot that we used `APIClientDecoder` for all decoding. Should be able to add a custom coding key to solve that issue though within the `GetClientIDResponse ` struct:
```
struct GetClientIDResponse: Decodable {

    private enum CodingKeys: String, CodingKey {
        case clientID = "clientId"
    }

    let clientID: String
}
```

Reply:
Ah nice that worked!